### PR TITLE
Add coding style about includes

### PR DESCRIPTION
**Pull request by LittleWhite-tb**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -25,6 +25,18 @@ Follow the indentation/whitespace style shown below. Do not use tabs, use 4-spac
 * For items that are both defined and declared in two separate files, put the doc-comment only next to the associated declaration. (In a header file, usually.) Otherwise, put it next to the implementation. Never duplicate doc-comments in both places.
 
 ```cpp
+// Includes should be in lexicographical sort
```
Comment (Subv) [inline CONTRIBUTING.md]: lexicographical order?

**Comment by LittleWhite-tb** [comment]
Updated

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -25,6 +25,18 @@ Follow the indentation/whitespace style shown below. Do not use tabs, use 4-spac
...
 ```cpp
+// Includes should be sorted lexicographically
+// STD includes first
+#include <map>
+#include <memory>
+
+// then, library includes
+#include <nihstro/shared_binary.h>
+
+// finally, citra includes
```
Comment (yuriks) [inline CONTRIBUTING.md]: We also tend to separate the includes of each major module ("common", "core", etc.) with a blank line between the groups.

**Comment by LittleWhite-tb** [comment]
Ok

**Comment by yuriks** [comment]
LGTM

**Comment by bunnei** [comment]
LGTM

**Pull request merged**